python/feast_spark/client.py

```diff
@@ -51,6 +47,45 @@
 )
 
 
+def stage_entities_to_bq_with_partition(
+    entity_source: pd.DataFrame, project: str, dataset: str
+) -> BigQuerySource:
+    """
+    Stores given (entity) dataframe as new table in BQ. Name of the table generated based on current time.
+    Table will expire in 1 day.
+    Returns BigQuerySource with reference to created table.
+    """
+    from google.cloud import bigquery
```

Comment:
what is the rationale for importing inside a func ?

I guess this is not expected to run in user's env and thus this dependency should be present in env ?

Is this method expected to be called rarely ?